apis/task.py

```diff
@@ -271,11 +268,107 @@ class GpuTask(BaseTask):
   """This is a class to set up tasks for GPU.
```

Comment:
Currently, TPU tasks are named as `TpuQueuedResourceTask` and `TpuXpkTask`. I did not think of a better name here. Your thoughts?

Reply:
I changed it to `GpuCreateResourceTask`. If we have a better name, I can make the update.